admin-dev/themes/new-theme/js/pages/product/components/dropzone/Dropzone.vue

```diff
@@ -308,7 +308,7 @@
         );
 
         this.sortableContainer.sortable({
-          items: DropzoneMap.sortableItems,
+          choices: DropzoneMap.sortableItems,
```

Comment:
Hmm, are you sure it's related? 🤔 Or is it an automatic change by your IDE maybe?
I don't see the connection with modified components

Reply:
stupid IDE changed too much things

Reply:
again, IDE was too "smart" did too much :D Ill revert this, I was only intending to change in CheckboxesDropdown